src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePicker.md

```diff
@@ -0,0 +1,25 @@
+# DatePicker
+
+## Summary
+
+DatePicker is use to select a specific date.
+
+*Button showing time open the time picker popup. 
+Bind to the Date property of the control to set initial time.
+*By default minute increment is set to 1
+*Cancel button cancel the new time selection. You can also click outside the time picker to do the same.
+*Done/OK button save the new selected time. 
```

Comment:
I think 2 lines don't apply to datepicker
-MinuteIncrement is N/A
- DatePicker doesn't have Cancel button in its default style (it should but doesn't yet)

Also should replace "time" with "date" :)